### DISCUSSION TITLE
Abandoned object detection?

### DISCUSSION THREAD

**Discussion by heyitsyang** [blakeblackshear/frigate#4175] [Ideas]
I'm new to frigate, so apologies if this has been covered in a discussion I haven't uncovered yet.

I'd like to detect packages on my from porch as an event.  For example, if a item of a certain size or larger is new to the frame and stationary for X minutes.

Is this something that has interest from other users?  Can this be done with the current software?

**Comment by NickM-27** [comment]
Frigate can not currently detect packages. It does have stationary detection and that info (how long the object has been stationary) is included in the MQTT events topic https://docs.frigate.video/integrations/mqtt#frigateevents
